### PR TITLE
Bootstrap test infrastructure and add a very basic test

### PR DESCRIPTION
**Pull request by browniebroke**
Following on:

- https://github.com/wharton/drf-excel/issues/86
- https://github.com/django-commons/membership/issues/45 
- https://github.com/orgs/django-commons/discussions/50

Add the test infrastructure using tox and github actions, taking inspiration from other projects in @django-commons

It might be beneficial to not add too many tests in this first PR, this way we should be able to get more folks to contribute in separate PRs to test different things. 

Let me know if the approach makes sense cc @tim-schilling in case you see something that goes against something in the org

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -3,7 +3,6 @@ name: CI
 on:
   pull_request:
   push:
-    branches: [main]
```
Comment (browniebroke) [inline .github/workflows/test.yml]: Removed this to see the result on the CI on my fork: https://github.com/browniebroke/drf-excel/actions

GitHub actions don't run new PR workflow added from a forks (I think)

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on:
+  pull_request:
+  push:
```
Comment (browniebroke) [inline .github/workflows/test.yml:5]: Perhaps better to add back before merging:

```suggestion
  push:
    branches: [main]
```

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
```
Comment (browniebroke) [inline .pre-commit-config.yaml:2]: I didn't add anything else like ruff yet as I'd like to limit the amount of code changes in here

File: tests/settings.py
Comment (browniebroke) [inline tests/settings.py:1]: I think that's the minimal setting file required for a project to use drf-excel

**Comment by FlipperPA** [comment]
This is great, thanks for getting this started, @browniebroke!

**Pull request merged**